bin/build.js

```diff
@@ -0,0 +1,75 @@
+const pkg = require('../package.json')
+const fs = require('fs')
+const icons = fs.readdirSync('lib').map((file) => {
+  if (file.slice(-4) !== '.svg') return // Skip non-icons
+  const id = file.slice(0, -4)
+  const code = String(fs.readFileSync(`lib/${file}`))
+  const [x, y, w, h] = String(code.match(/viewBox="[^"]+/)).split(' ').map(Number)
+  const body = code.replace(/^[^>]+>|<[^<]+$/g, '').replace(/\s*([<>])\s*/g, '$1') // Strip white space around tokens
+  const svg = `<svg style="width:${w / 10}em;height:${h / 10}em" viewBox="0 0 ${w} ${h}" focusable="false" aria-hidden="true">${body}</svg>`
+  const key = id.replace(/-+./g, (m) => m.slice(-1).toUpperCase()) // camelCase
+  const jsx = key.replace(/./, (m) => m.toUpperCase()) // TitleCamelCase
+  return { id, body, key, jsx, svg, x, y, w, h }
+}).filter(Boolean) // Remove falsy values
+
+generateSketch()
+generateDocs()
+generateJSON()
+generateIIFE()
+generateMJS()
+generateCJS()
+generateJSX()
+
+function generateJSON () {
+  fs.writeFileSync('lib/core-icons.json', JSON.stringify(icons.map(({ id }) => `${id}.svg`)))
+}
+
+function generateCJS () {
+  fs.writeFileSync('core-icons.cjs', `module.exports = {${icons.map(({ key, svg }) => `'${key}':'${svg}'`).join(',')}}`)
```

Comment:
`.cjs` is a non-standard extension. Should probably rename to `.js`
```suggestion
  fs.writeFileSync('core-icons.js', `module.exports = {${icons.map(({ key, svg }) => `'${key}':'${svg}'`).join(',')}}`)
```